pyproject.toml

```diff
@@ -7,8 +7,8 @@ name = "aiida_test_cache"
 
 [project]
 name = "aiida-test-cache"
+version = "0.0.1a0"
```

Comment:
I decided to first publish a version `0.0.1` which should be fully backwards compatible and can be used by existing users.